BUILDING.md

```diff
@@ -387,9 +387,9 @@ Prerequisites:
 * [Python 2.6 or 2.7](https://www.python.org/downloads/)
 * The "Desktop development with C++" workload from
   [Visual Studio 2017](https://www.visualstudio.com/downloads/) or the
-  "Visual C++ build tools" workload from the
+  "Build Tools for Visual Studio 2017" workload from the
   [Build Tools](https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2017),
-  with the default optional components.
+  with the default optional components.  Once the "Build Tools for Visual Studio 2017" downloaded, install it by choosing 'Visual C++ build tools' from the installer and proceed.  It will install the necessary build tools to compile the c++ files.
```

Comment:
Thank you!
Just some formatting and consistency fixing:
* Wrap long lines at 80 characters maximum.
* Delete extra spaces.
* Double quotes instead of single quotes.
* Uppercase C++.

You can apply these fixes by accepting this suggestion:

```suggestion
  with the default optional components. Once the "Build Tools for Visual Studio
  2017" downloaded, install it by choosing "Visual C++ build tools" from the
  installer and proceed. It will install the necessary build tools to compile
  the C++ files.
```

Reply:
@vsemozhetbyt - Thank you for the review and help. I will remember these points next time.